### PR TITLE
added options to include vertical filter styles for bef.

### PR DESCRIPTION
**Pull request by Krystation**
Not a plugin but adding in options to the settings, Includes some css and js.

## Description

Styles for the vertical filters. Also adds an option to move the reset button above everything else. JS in there sets a class on li that contain other list elements. 

changes the arrow to more of the chevron, font color changes, 

## Related issues

#2275 

## How to test

Enable in extend. In view, options should show in advanced

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_bef_extend/js/az-bef-extend.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+(function (Drupal) {
```
Comment (tadean) [inline modules/custom/az_bef_extend/js/az-bef-extend.js:1]: Javascript for this should probably be written as ES6 and transpiled to ES5 via our [javascript transpilation process](https://github.com/az-digital/az_quickstart/blob/main/CONTRIBUTING.md#compiling-javascript-in-local-development)

File: modules/custom/az_bef_extend/js/az-bef-extend.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  Drupal.behaviors.myBehavior = {
```
Comment (tadean) [inline modules/custom/az_bef_extend/js/az-bef-extend.js:2]: It might be good to choose a more specific identifier for this behavior name.

**Comment by trackleft** [comment]
FYI
You should be able to fix these by running `lando phpcbf`

```
114FILE: .../az_quickstart/modules/custom/az_bef_extend/az_bef_extend.module
115----------------------------------------------------------------------
116FOUND 32 ERRORS AND 4 WARNINGS AFFECTING 31 LINES
117----------------------------------------------------------------------
118   1 | ERROR   | [x] Missing file doc comment
119     |         |     (Drupal.Commenting.FileComment.Missing)
120   4 | WARNING | [x] Unused use statement
121     |         |     (Drupal.Classes.UnusedUseStatement.UnusedUse)
122   6 | ERROR   | [x] Missing function doc comment
123     |         |     (Drupal.Commenting.FunctionComment.Missing)
124   7 | ERROR   | [x] Inline comments must end in full-stops,
125     |         |     exclamation marks, question marks, colons, or
126     |         |     closing parentheses
127     |         |     (Drupal.Commenting.InlineComment.InvalidEndChar)
128  15 | ERROR   | [x] Inline comments must end in full-stops,
129     |         |     exclamation marks, question marks, colons, or
130     |         |     closing parentheses
131     |         |     (Drupal.Commenting.InlineComment.InvalidEndChar)
132  16 | WARNING | [ ] You are using the comparison operator "==" that
133     |         |     converts type and may cause unintended
134     |         |     results.
135     |         |     (Security.Misc.TypeJuggle.TypeJuggle)
136  17 | ERROR   | [x] Data types in @var tags need to be fully
137     |         |     namespaced
138     |         |     (Drupal.Commenting.DataTypeNamespace.DataTypeNamespace)
139  20 | ERROR   | [x] Expected 1 space after IF keyword; 0 found
140     |         |     (Drupal.ControlStructures.ControlSignature.SpaceAfterKeyword)
141  20 | WARNING | [ ] You are using the comparison operator "!=" that
142     |         |     converts type and may cause unintended
143     |         |     results.
144     |         |     (Security.Misc.TypeJuggle.TypeJuggle)
145  20 | ERROR   | [x] Expected 1 space after closing parenthesis;
146     |         |     found 0
147     |         |     (Drupal.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis)
148  23 | ERROR   | [x] Whitespace found at end of line
149     |         |     (Squiz.WhiteSpace.SuperfluousWhitespace.EndLine)
150  26 | ERROR   | [x] Whitespace found at end of line
151     |         |     (Squiz.WhiteSpace.SuperfluousWhitespace.EndLine)
152  27 | ERROR   | [x] Inline comments must end in full-stops,
153     |         |     exclamation marks, question marks, colons, or
154     |         |     closing parentheses
155     |         |     (Drupal.Commenting.InlineComment.InvalidEndChar)
156  31 | ERROR   | [x] Whitespace found at end of line
157     |         |     (Squiz.WhiteSpace.SuperfluousWhitespace.EndLine)
158  32 | ERROR   | [x] Inline comments must end in full-stops,
159     |         |     exclamation marks, question marks, colons, or
160     |         |     closing parentheses
161     |         |     (Drupal.Commenting.InlineComment.InvalidEndChar)
162  42 | ERROR   | [x] Inline comments must end in full-stops,
163     |         |     exclamation marks, question marks, colons, or
164     |         |     closing parentheses
165     |         |     (Drupal.Commenting.InlineComment.InvalidEndChar)
166  43 | WARNING | [ ] You are using the comparison operator "==" that
167     |         |     converts type and may cause unintended
168     |         |     results.
169     |         |     (Security.Misc.TypeJuggle.TypeJuggle)
170  45 | ERROR   | [x] Expected 1 space after IF keyword; 0 found
171     |         |     (Drupal.ControlStructures.ControlSignature.SpaceAfterKeyword)
172  45 | ERROR   | [x] Expected 1 space after closing parenthesis;
173     |         |     found 0
174     |         |     (Drupal.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis)
175  47 | ERROR   | [x] Whitespace found at end of line
176     |         |     (Squiz.WhiteSpace.SuperfluousWhitespace.EndLine)
177  48 | ERROR   | [x] Inline comments must end in full-stops,
178     |         |     exclamation marks, question marks, colons, or
179     |         |     closing parentheses
180     |         |     (Drupal.Commenting.InlineComment.InvalidEndChar)
181  50 | ERROR   | [x] Whitespace found at end of line
182     |         |     (Squiz.WhiteSpace.SuperfluousWhitespace.EndLine)
183  51 | ERROR   | [x] Inline comments must end in full-stops,
184     |         |     exclamation marks, question marks, colons, or
185     |         |     closing parentheses
186     |         |     (Drupal.Commenting.InlineComment.InvalidEndChar)
187  56 | ERROR   | [x] Inline comments must end in full-stops,
188     |         |     exclamation marks, question marks, colons, or
189     |         |     closing parentheses
190     |         |     (Drupal.Commenting.InlineComment.InvalidEndChar)
191  57 | ERROR   | [x] TRUE, FALSE and NULL must be uppercase; expected
192     |         |     "FALSE" but found "false"
193     |         |     (Generic.PHP.UpperCaseConstant.Found)
194  60 | ERROR   | [x] Line indented incorrectly; expected 8 spaces,
195     |         |     found 10
196     |         |     (Drupal.WhiteSpace.ScopeIndent.IncorrectExact)
197  60 | ERROR   | [x] TRUE, FALSE and NULL must be uppercase; expected
198     |         |     "FALSE" but found "false"
199     |         |     (Generic.PHP.UpperCaseConstant.Found)
200  61 | ERROR   | [x] Line indented incorrectly; expected 10 spaces,
201     |         |     found 12
202     |         |     (Drupal.WhiteSpace.ScopeIndent.IncorrectExact)
203  61 | ERROR   | [x] TRUE, FALSE and NULL must be uppercase; expected
204     |         |     "TRUE" but found "true"
205     |         |     (Generic.PHP.UpperCaseConstant.Found)
206  62 | ERROR   | [x] Line indented incorrectly; expected 10 spaces,
207     |         |     found 12
208     |         |     (Drupal.WhiteSpace.ScopeIndent.IncorrectExact)
209  63 | ERROR   | [x] Line indented incorrectly; expected 8 spaces,
210     |         |     found 10
211     |         |     (Drupal.WhiteSpace.ScopeIndent.IncorrectExact)
212  68 | ERROR   | [x] Inline comments must end in full-stops,
213     |         |     exclamation marks, question marks, colons, or
214     |         |     closing parentheses
215     |         |     (Drupal.Commenting.InlineComment.InvalidEndChar)
216  85 | ERROR   | [x] Inline comments must end in full-stops,
217     |         |     exclamation marks, question marks, colons, or
218     |         |     closing parentheses
219     |         |     (Drupal.Commenting.InlineComment.InvalidEndChar)
220 100 | ERROR   | [x] Inline comments must end in full-stops,
221     |         |     exclamation marks, question marks, colons, or
222     |         |     closing parentheses
223     |         |     (Drupal.Commenting.InlineComment.InvalidEndChar)
224 104 | ERROR   | [x] Inline comments must end in full-stops,
225     |         |     exclamation marks, question marks, colons, or
226     |         |     closing parentheses
227     |         |     (Drupal.Commenting.InlineComment.InvalidEndChar)
228 106 | ERROR   | [x] Object operator not indented correctly; expected
229     |         |     4 spaces but found 12
230     |         |     (Drupal.WhiteSpace.ObjectOperatorIndent.Indent)
231----------------------------------------------------------------------
232PHPCBF CAN FIX THE 33 MARKED SNIFF VIOLATIONS AUTOMATICALLY
233----------------------------------------------------------------------
```

File: package-lock.json
Comment (trackleft) [inline package-lock.json:1]: This file should also be deleted.

File: modules/custom/az_bef_extend/css/az-bef-extend.scss [hunk 3]
```diff
@@ -0,0 +1,96 @@
+.bef-exposed-form{
```
[the remark this answers is not in the document]
Comment (Krystation) [reply modules/custom/az_bef_extend/css/az-bef-extend.scss:1]: lol, just live sass compiler on vscode

**Comment by danahertzberg** [comment]
Can/should this PR be deleted in favor of https://github.com/az-digital/az_quickstart/pull/2998?

**Comment by Krystation** [comment]
> Can/should this PR be deleted in favor of https://github.com/az-digital/az_quickstart/pull/2998?

Yeee, delete away

**Pull request closed without merging**